### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by entertainment-veks**

### CODE REVIEW

File: src/main/java/core/basesyntax/model/Bet.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.model;
+
+public class Bet {
+    int value;
```
Comment (Ketch92) [inline src/main/java/core/basesyntax/model/Bet.java:4]: the task requires to add another model (User as an example) and Dao implementation for it.

**Review by Ketch92 — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BoxDao;
+import core.basesyntax.dao.BoxDaoImpl;
+import core.basesyntax.model.Box;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BoxDao boxDao = new BoxDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:9]: You should have two models and two dao classes for each of them at the result

Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java:9]: access modifiers

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            try {
+                String[] boxData = command.split(" ");
+                int height = Integer.parseInt(boxData[0]);
+                int weight = Integer.parseInt(boxData[1]);
+                box = new Box(height, weight);
+            } catch (NumberFormatException | ArrayIndexOutOfBoundsException e) {
+                System.out.println("Please, enter correct data");
+            }
+
+            boxDao.add(box);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:31]: let's add only if dao not null and use method getAll() to print all values

File: src/main/java/core/basesyntax/model/Box.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.model;
+
+public class Box {
+    int height;
+    int weight;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/model/Box.java:5]: access modifiers

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+public class ConsoleHandler {
+    private BoxDao boxDao = new BoxDaoImpl();
+    private UserDao userDao = new UserDaoImpl();
+    private Scanner scanner = new Scanner(System.in);
+
+    public void handle() {
+
+        while (true) {
+            System.out.println("Which object do you want to add?");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:19]: would be good to print all options for user

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

**Review by dmytro-hryhoruk — APPROVED** [review]
done for approve

**Pull request merged**